.github/workflows/README.md

```diff
@@ -0,0 +1,27 @@
+# GHA for Psi4
+
+## [docs.yml](./docs.yml)
+
+* Since: April 2021
+* Runs on: push to master
+* Goals:
+  * get CI warning if doxygen docs compile broken
+  * get CI warning if sphinx docs compile broken 🍏, or compile with warnings 🍏, or compile with unreachable links ❌, or can't compile nit-picky ❌
```

Comment:
Why is the fruit here?

Reply:
I copied this bit from an unmerged PR from last year. As far as I remember, green apple was the compromise for something approving/green that showed up on dark and light themes.

Reply:
Okay, so what controls whether we have a green apple or a red x for an item?

Reply:
green apple is assured by passing `-D SPHINXMAN_STRICT=ON` to the gha docs build and then that invokes `-W` to prompt us to address warnings. https://github.com/psi4/psi4/blob/master/doc/sphinxman/CMakeLists.txt#L35-L36

Red "x"s can't be green apples until turn on nit-picky flag. And that can't be until every fn in the driver is addressable, https://github.com/psi4/psi4/pull/2166 .

Reply:
So green apple = this is something that CI will check, and red X = this is something _we would like_ CI to check but it doesn't right now? If so, please make that explicit.